classes/class-wc-klarna-order-management-order-lines.php

```diff
@@ -22,42 +22,42 @@ class WC_Klarna_Order_Management_Order_Lines {
 	/**
 	 * Klarna order order lines.
 	 *
-	 * @var $order_lines
+	 * @var array
 	 */
 	public $order_lines = array();
 
 	/**
 	 * Klarna order amount.
 	 *
-	 * @var $order_lines
+	 * @var integer
 	 */
 	public $order_amount = 0;
 
 	/**
 	 * Klarna order tax amount.
 	 *
-	 * @var $order_tax_amount
+	 * @var integer
 	 */
 	public $order_tax_amount = 0;
 
 	/**
 	 * WooCommerce order ID.
 	 *
-	 * @var $order_id
+	 * @var string
```

Comment:
WooCommerce order IDs are ints generally. Is it a string here?

Reply:
The code this was originally called from, in the now removed includes/class-wc-klarna-order-management-request.php, it was declared as a string.
The same value now is in the KOM_Request class, and in trying to keep it as close to the original code as possible I declared it as string there as well.

Tracing it back through the request structure, into WooCommerce and out the other end, yes, it seems to be an int in the very bottom.

I'll update it _everywhere_ to reflect this.